h/services/bulk_annotation.py

```diff
@@ -110,10 +113,17 @@ def annotation_search(
     def _search_query(cls, authority, audience, created) -> Select:
         """Generate a query which can then be executed to find annotations."""
         return (
-            sa.select([cls._AUTHOR.username, Group.authority_provided_id])
+            sa.select(
+                [
+                    cls._AUTHOR.username,
+                    Group.authority_provided_id,
+                    sa.func.coalesce(AnnotationMetadata.data, "{}").label("metadata"),
```

Comment:
Defaulting to `{}` I reckon that's a bit more usable for callers than null?

ie doing anno.metadata.get('assignmet_id') instead of checking for none before?

Not a big preference one way or the other.

Reply:
So `BulkAnnotationService` is now querying (and joining to) the `AnnotationMetadata` table as well as `User` and `Group`